app/controllers/concerns/course/activities_concern.rb

```diff
@@ -0,0 +1,16 @@
+module Course::ActivitiesConcern
+  extend ActiveSupport::Concern
+
+  # Load recent activity feeds
+  #
+  # @param [Course] course course which activities are related to
+  # @param [Integer] number number of activities that will be loaded
+  #
+  # @return [Array<Activity>]
+  def recent_activities(course, number: nil)
+    result = Activity.joins(:notification).where('notifications.activity_feed = true').
```

Comment:
Wait. Seems you are right. Cool.. But why here should use `where(notifications:` not `where(notification:` @lowjoel